tests/property/ic_api/candid/test/tests.ts

```diff
@@ -1,110 +1,78 @@
-import { ActorSubclass } from '@dfinity/agent';
-import { defaultPropTestParams, expect, it, Test } from 'azle/test';
+globalThis._azleExperimental = true;
+import { execSyncPretty } from 'azle/src/build/stable/utils/exec_sync_pretty';
+import { DidVisitor, getDefaultVisitorData } from 'azle/src/lib/stable';
+import {
+    defaultPropTestParams,
+    expect,
+    getCanisterActor,
+    it,
+    please,
+    Test
+} from 'azle/test';
+import { candidDefinitionArb } from 'azle/test/property/arbitraries/candid/candid_definition_arb';
+import { DefinitionConstraints } from 'azle/test/property/arbitraries/candid/candid_definition_arb/types';
+import { execSync } from 'child_process';
 import fc from 'fast-check';
-import { TextDecoder, TextEncoder } from 'util';
 
-// @ts-ignore this path may not exist when these tests are imported into other test projects
-import { _SERVICE } from './dfx_generated/canister/canister.did';
+import { _SERVICE as Actor } from './dfx_generated/canister/canister.did';
 
-export function getTests(canister: ActorSubclass<_SERVICE>): Test {
+export function getTests(): Test {
     return () => {
-        it('should encode and decode Candid strings correctly, with UTF-8 verification', async () => {
-            await fc.assert(
-                fc.asyncProperty(fc.string(), async (arbitraryString) => {
-                    const candidString = toCandidString(arbitraryString);
-
-                    const encodedBytes = await checkCandidEncoding(
-                        canister,
-                        arbitraryString,
-                        candidString
-                    );
-
-                    const decodedString = await checkCandidDecoding(
-                        canister,
-                        encodedBytes
-                    );
-
-                    expect(decodedString).toBe(candidString);
-                }),
-                defaultPropTestParams
+        please('install didc', async () => {
+            execSync(
+                `cargo install --git https://github.com/dfinity/candid --rev 5d3c7c35da652d145171bc071ac11c63d73bf803 --force didc`,
+                { stdio: 'inherit' }
             );
         });
 
-        it('should decode Candid bytes and then encode correctly, with UTF-8 verification', async () => {
+        it('should encode and decode candid values correctly', async () => {
```

Comment:
Let's add a comment into this file explaining how to make the tests better, explaining the limitations, explaining that we could add arbitrary candid string creation to our candid arbitraries

Reply:
Done